src/main/java/binary_search/OccuranceOfElement.java

```diff
@@ -0,0 +1,37 @@
+package binary_search;
+    public class OccuranceOfElement {
+        public int countOfElement(int[] nums,int target){
+            int start=0;
+            int end=nums.length-1;
+            int mid;
+            while(start<=end){
+                mid=(start+end)/2;
+                if(nums[mid]==target)
+                    return getCount(nums,mid,target);
```

Comment:
This method name do not describe what it is trying to do. So abstracting the logic behind a mthod is perfect, but the name should be more descriptive. Let me think of a name for this.